libCacheSim/bin/cachesim/cache_init.h

```diff
@@ -138,6 +138,14 @@ static inline cache_t *create_cache(const char *trace_path, const char *eviction
     cache = QDLP_init(cc_params, eviction_params);
   } else if (strcasecmp(eviction_algo, "sieve") == 0) {
     cache = Sieve_init(cc_params, eviction_params);
+#ifdef ENABLE_3L_CACHE
```

Comment:
We want to ensure that `eviction_params` in the code are not overwritten (like LRB), as this could invalidate inputs provided via the `-e` option. To maintain clarity and flexibility, we can map `-e "objective=object-miss-ratio"` to call `3LCache-OMR`, while allowing `3LCache` to be called by default when no eviction parameters are specified.